package.json

```diff
@@ -18,12 +18,12 @@
   ],
   "scripts": {
     "test": "exit 0",
-    "build": "rollup -cm -n picodom -f umd -i index.js -o dist/picodom.js",
-    "format": "prettier --semi false --write index.js"
+    "build": "rollup --config --format umd --input index.js --name picodom --output dist/picodom.js --sourcemap",
```

Comment:
My thought was that the longer form was easier to read. I can just scrap this PR.